src/drt/src/gc/FlexGC_impl.h

```diff
@@ -218,6 +218,14 @@ class FlexGCWorker::Impl
   void initRegionQuery();
 
   void checkMetalSpacing();
+  void checkMetalSpacing_wrongDir_getQueryBox(gcSegment* edge,
+                                              frCoord spcVal,
+                                              box_t& queryBox);
+  gtl::orientation_2d getOrientation(gcSegment* edge) const;
```

Comment:
Good idea to have this as a method, But I think it should be a public method for gcSegment

Reply:
I moved getOrientation() inside gcSegment as noted.
Did you mean the same for checkMetalSpacing_wrongDir_getQueryBox() too?